src/Raven.Server/Documents/StreamsTempFile.cs

```diff
@@ -170,11 +170,13 @@ public IDisposable CreateReaderStream(out LimitedStream limitedStream)
                 var safeFileHandle = new SafeFileHandle(_parent._file.InnerStream.SafeFileHandle.DangerousGetHandle(), ownsHandle: false);
                 Stream stream = new FileStream(safeFileHandle, FileAccess.Read);
                 stream.Seek(_startPosition, SeekOrigin.Begin);
+                var relativePosition = _startPosition;
                 if (_stream is TempCryptoStream tcs)
                 {
+                    relativePosition = 0;
```

Comment:
do we have a test for this case?

Reply:
yes, the `EncryptedBackupAndRestoreShardedDatabase..` tests